backend/avatar/urls.py

```diff
@@ -13,11 +13,13 @@
     1. Import the include() function: from django.urls import include, path
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
+from avatarcore.views import landing_page
 from django.contrib import admin
 from django.urls import path, include
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-
     path('auth/', include('avatarauth.urls', namespace='auth')),
+    path('', landing_page, name='landing-page'),
```

Comment:
Landing page should not be served by django! It's a simple static html which will be served by the nginx only.